hstream/src/HStream/Server/Handler.hs

```diff
@@ -432,7 +432,7 @@ deleteSubscriptionHandler :: ServerContext
 deleteSubscriptionHandler ServerContext{..} (ServerNormalRequest _metadata DeleteSubscriptionRequest{..}) = do
   hm <- readIORef subscribedReaders
   case HM.lookup deleteSubscriptionRequestSubscriptionId hm of
-    Nothing                         -> return $ ServerNormalResponse Empty [] StatusOk ""
+    Nothing                         -> return $ ServerNormalResponse (Just Empty) [] StatusOk ""
```

Comment:
why not ServerNormalResponse Nothing?
But is it follow the google cloud api design pattern, use Maybe instead of Empty?

Reply:
The main reason for this change is that the original grpc-haskell library requires that the ServerNormalResponse must return a response, but when the rpc method fails, or if the call does not complete properly, there is no suitable response to return. Even if the response is returned, it is a meaningless value. So the original ServerNormalResponse has been modified to return a Nothing when the above failure occurs. Also, there is a bug in the original grpc-haskell implementation and It causes the client to receive both success and failure at the same time.